### ISSUE TITLE
CP-SAT solver: large_jobs option for scheduling_with_transaction example 

### ISSUE THREAD

**Issue by YooTimmy** [google/or-tools#1585]
Hello everyone,

I am trying to study the sample code for [scheduling_with_transition.py](https://github.com/google/or-tools/blob/stable/examples/contrib/scheduling_with_transitions_sat.py
).
I am able to run with small_jobs option without issue. But for large_jobs option, I waited for very long (1 hour?) since it started running, but seems I could not get any output. 

I am now wondering it this is expected? I tried to search in forums but seems no similar questions posted. Maybe I missed something. 

Thank you for helping out, and sorry for this simple question..

**Comment by stradivari96** [comment]
No output?
```
python scheduling_with_transitions_sat.py --problem_instance 1
```
Returns on my machine
```
Horizon = 989
Solution 0, time = 2.569877 s, objective = 719, makespan = 559
Solution 1, time = 3.077210 s, objective = 703, makespan = 553
```
And prints the solution if I stop it with CTRL-C.

You can also run it with `--params num_search_workers:8`

**Comment by YooTimmy** [comment]
Hello Xiang Chen,

Thanks for getting back!

On my machine when I tried `python scheduling_with_transitions_sat.py --problem_instance 1`,
seems nothing was returned for quite some time. But when I added `num_search_workers:8`, at least I can get first solution in 8s. Then the next solution came out only after waiting for 6-7 minutes..
However seems on your machine it is performing much better than mine. Any idea what might cause this difference? In case it is related to the hardware specs, my laptop has 24 GB RAM and Intel i7-7700HQ CPU @2.80GHz.

![image](https://user-images.githubusercontent.com/32185776/65202536-063f5a00-dabd-11e9-8d83-ee5b8efee80b.png)

**Comment by stradivari96** [comment]
Hmm, which version of ortools are you using?

**Comment by lperron** [comment]
indeed, the examples has moved from examples/python to examples/contrib a
few versions ago.
Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00



Le jeu. 19 sept. 2019 à 09:46, Xiang Chen <notifications@github.com> a
écrit :

**Comment by YooTimmy** [comment]
Hello Xiang Chen and Laurent,

Sorry for later reply.
And you are right! previously I was using version 7.2.6977, and after I updated it to the latest version google OR-Tools, seems I can get similar response as yours. Though still I think getting the solutions after the first several output takes some time, I guess this is expected? (I waited like 15 mins and no further output yet, as attached.) I will pay attention to update the library in the future. 
![image](https://user-images.githubusercontent.com/32185776/65292823-7ddbba80-db8b-11e9-89f5-5fcbc66f19d3.png)

And actually I just released that some examples are inside examples/contrib folder, previously I am only looking for the samples in python folder. Will pay attention to that as well.
